gosu-test/src/test/gosu/gw/specContrib/statements/usesStatement/Errant_OnDemandStaticImportsAllowedByUnqualifiedClassNameFromTheSamePackage.gs

```diff
@@ -0,0 +1,11 @@
+package gw.specContrib.statements.usesStatement
```

Comment:
Same here. If the file is correct, it should not have an Errant_ prefix in the name.